src/Engine/Graphics/Renderer/OpenGLRenderer.cpp

```diff
@@ -2597,10 +2596,9 @@ void OpenGLRenderer::DrawBillboards() {
     billbshader.use();
 
     // set sampler to palette
-    glUniform1i(billbshader.uniformLocation("palbuf"), GLint(1));
-    glActiveTexture(GL_TEXTURE0 + 1);
-    glBindTexture(GL_TEXTURE_BUFFER, paltex);
-    glTexBuffer(GL_TEXTURE_BUFFER, GL_RGBA8UI, palbuf);
+    glBindTexture(GL_TEXTURE_2D, paltex2D);
```

Comment:
glActiveTexture(GL_TEXTURE0 + paltex2D_id);
glBindTexture(GL_TEXTURE_2D, paltex2D);

need to select the correct texture unit then bind to it here